ibm_catalog.json

```diff
@@ -65,34 +66,43 @@
               "role_crns": [
                 "crn:v1:bluemix:public:iam::::role:Administrator"
               ],
-              "service_name": "all-account-management-services"
+              "service_name": "all-account-management-services",
+              "notes": "Required for Account Management Services."
             },
             {
               "role_crns": [
                 "crn:v1:bluemix:public:iam::::role:Editor"
               ],
-              "service_name": "databases-for-redis"
+              "service_name": "databases-for-redis",
+              "notes": "Required for working with Databases for Redis."
```

Comment:
You don't need to add a note for every required permissions - just the optional ones as to why when you would need it